### ISSUE TITLE
replace_word_elongations creates NAs

### ISSUE THREAD

**Issue by erhard1** [trinker/textclean#61]
I was doing some text clean up with replace_word_elongations and ran into a case where the phrase "AAA battery" caused all word elongation matches to become NA. I modified the example from the help file to show the situation:

x <- c('look', 'noooooo!', 'real coooool!', "it's sooo goooood", 'fsdfds',
       'fdddf', 'as', "aaaahahahahaha", "aabbccxccbbaa", 'I said heyyy!',
       "I'm liiiike whyyyyy me?", "Wwwhhatttt!", "AAA battery")

replace_word_elongation(x)
 [1] "look"          NA              NA              NA              "fsdfds"        NA              "as"            NA             
 [9] "aabbccxccbbaa" NA              NA              NA              NA   

Thanks in advance for any help you can provide. I have been pleased with how the textclean package has been working so far.

**Comment by ihsankahveci** [comment]
Assuming you are using the CRAN version, I wanted to let you know that this issue should be solved after the last commit: https://github.com/trinker/textclean/commit/5443d7484cd798e7494a63f794e6fb30834e5ec2.

You can install  the latest version using `pacman` package:
```r
pacman::p_load_gh(
    "trinker/lexicon",    
    "trinker/textclean"
)
```
Or using `remotes` package:
```r
remotes::install_github("trinker/textclean", force = TRUE)
```

**Comment by erhard1** [comment]
Yes, I am using the current version on CRAN. I will give that new commit a try. The work around I used was to transform all the text to lower case. That resolved the issue I was having wit moved case.
